contracts/handlers/ERC20Handler.sol

```diff
@@ -92,24 +92,131 @@ contract ERC20Handler is IDepositHandler, ERC20Safe {
         );
     }
 
-    function createTokenID(uint256 chainID, address originChainTokenAddress) internal returns (string memory) {
+    function createTokenID(uint256 chainID, address originChainTokenAddress) internal pure returns (string memory) {
         return string(abi.encodePacked(chainID, originChainTokenAddress));
     }
 
-    // TODO If any address can call this, anyone can mint tokens
+    // Copied from https://github.com/provable-things/ethereum-api/blob/master/oraclizeAPI_0.5.sol#L1020
+    function parseInt(string memory _a, uint _b) internal pure returns (uint _parsedInt) {
```

Comment:
push this to a helper file?

Reply:
Forgot to do this, will push all the helper methods into a file: `helpers/StringUtils.sol`